lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php

```diff
@@ -742,7 +738,7 @@ public function getMigrationsToExecute($direction, $to)
         }
 
         if ($direction === Version::DIRECTION_DOWN) {
-            if (count($this->migrations)) {
+            if ($this->migrations) {
```

Comment:
I prefer to be as much explicit as possible as to what is tested in a conditional.
What is the advantage of removing the count ?
@ReenExe